package.json

```diff
@@ -84,6 +86,7 @@
     "@types/classnames": "2.2.11",
     "@types/dateformat": "3.0.1",
     "@types/electron-devtools-installer": "2.2.0",
+    "@types/fs-extra": "9.0.11",
```

Comment:
We don't need this package

Reply:
Thanks for your comment.
We need to use this "@types/fs-extra": "9.0.11" package. 
Now we are using fs-extra. I tried to use fs but it doesn't work. it makes the malformed error. And they recommended using fs-extra. and you know, "@types/fs-extra": "9.0.11" package didn't be provided as a default, so we need it.
I tried to remove this package but got this error message.
```
Could not find a declaration file for module 'fs-extra'. 'D:/git/pastel-electron-wallet/node_modules/fs-extra/lib/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/fs-extra` if it exists or add a new declaration (.d.ts) file containing `declare module 'fs-extra';`
```
Please check it again and let me know your opinion.
Thanks

Reply:
> I tried to use fs but it doesn't work

What is the detail error of usage of original `fs`?

Reply:
I got following error when I use fs and keep the db data.

```
await fs.promises.writeFile(
    path.join(remote.app.getPath('appData'), 'Pastel', 'pasteldb.sqlite'),
    buffer,
    { flag: 'a+' },
  )
```
`This file is malformed.`

I think this file is corrupt, but fs-extra does work well.